### ISSUE TITLE
Please check if you are train learning properly.

### ISSUE THREAD

**Issue by ljj727** [costapt/adversarial_retinal_synthesis#8]
Hi, I am having trouble using the code.  It was confirmed that the loss of the code discriminator seems strange during the learning process. The result is as follows, which part is the problem? Thank you.

![5](https://user-images.githubusercontent.com/62738682/79180511-22358f00-7e45-11ea-9787-78dde4fb149f.PNG)
![7](https://user-images.githubusercontent.com/62738682/79180513-2366bc00-7e45-11ea-9cfb-2a7025ac4879.PNG)
![6](https://user-images.githubusercontent.com/62738682/79180515-23ff5280-7e45-11ea-8ba1-77d12a8c9cbc.PNG)
![1](https://user-images.githubusercontent.com/62738682/79180516-25307f80-7e45-11ea-818a-d0d047cf3832.PNG)
![2](https://user-images.githubusercontent.com/62738682/79180517-25307f80-7e45-11ea-8980-5f8b4dbd7708.PNG)
![3](https://user-images.githubusercontent.com/62738682/79180518-25c91600-7e45-11ea-9560-5691c06e358d.PNG)
![4](https://user-images.githubusercontent.com/62738682/79180519-25c91600-7e45-11ea-903b-e38b590ae271.PNG)

![8](https://user-images.githubusercontent.com/62738682/79180629-63c63a00-7e45-11ea-82f9-683c10caf859.PNG)

**Comment by AliSaeed86** [comment]
Hi @kaejong2,
kindly, if you were able to execute the code with keras 2.x and python 3.x, it would be very kind of you to share the code with me. actually i am facing difficulties doing the required changes.

Please, email to ali.1986.kasim@gmail.com
I would be so thankful to you.

Regards.

**Comment by FFjiahao** [comment]
@AliSaeed86  Can you run with the author's code? I can run the author's code but it doesn't work well.How does your code work？
